packages/frontend-2/lib/projects/composables/projectManagement.ts

```diff
@@ -133,6 +137,37 @@ export function useCreateProject() {
               },
               { fieldNameWhitelist: ['admin'] }
             )
+
+            // New cache update for Workspace.projects
+            if (input.workspaceId) {
+              const workspaceCacheId = getCacheId('Workspace', input.workspaceId)
+
+              modifyObjectFields<
+                { filter?: { search?: string } },
```

Comment:
You should use GQLGen types, instead of typing out these types manually, otherwise if they change this code is not gonna show errors.

`modifyObjectFields<WorkspaceProjectsArgs, Workspace['projects']>`